✨ Increase cbBTC supply caps on Ethereum main and Base (#478)
bgd-labs/aave-proposals-v3: src/20241004_Multi_IncreaseCbBTCSupplyCaps/IncreaseCbBTCSupplyCaps.md

---
title: "Increase cbBTC Supply Caps"
author: "ACI"
discussions: "https://governance.aave.com/t/arfc-increase-cbbtc-supply-caps-on-aave-v3-ethereum-market-and-base/19304"
---

## Simple Summary

This proposal aims to increase the supply caps for cbBTC on both the Aave V3 Ethereum Market and BASE. The increase in supply caps will allow for greater utilization of cbBTC within the Aave ecosystem, potentially improving liquidity and capital efficiency.

## Motivation

The current supply caps for cbBTC on Aave V3 Ethereum and BASE markets may be limiting the potential growth and utilization of this asset, as we’ve seen demand and current Supply Caps not being enough to allocate it.

By increasing these caps, we can:

1. Enhance liquidity for cbBTC within the Aave protocol.
2. Provide more opportunities for users to supply and borrow cbBTC.
3. Potentially attract more users and capital to the Aave ecosystem.

## Specification

Ticker: cbBTC (cbBTC)

> Mainnet: 0x5c647cE0Ae10658ec44FA4E11A51c96e94efd1Dd
> 
> Base: 0xBdb9300b7CDE636d9cD4AFF00f6F009fFBBc8EE6

Reviewer: ```suggestion
Mainnet: 0xcbB7C0000aB88B473b1f5aFd9ef808440eed33Bf

Base: 0xcbB7C0000aB88B473b1f5aFd9ef808440eed33Bf
```

Author: Interesting, why the generator put the aToken instead of the underlying?
Maybe a bug somewhere, I can check if I find the time to

> We propose to Increase Supply Caps on Mainnet to 3000 cbBTC and to Increase Supply Caps on Base to 1700 cbBTC.

Reviewer: ```suggestion
We propose to Increase Supply Caps on Mainnet to 10000 cbBTC and to Increase Supply Caps on Base to 5000 cbBTC.
```

| cbBTC                | Ethereum V3 | Base       |
| -------------------- | ----------- | ---------- |

> | Increase Supply Caps | 3000 cbBTC  | 1700 cbBTC |

Reviewer 2: ```suggestion
| Increase Supply Caps | 10000 cbBTC  | 5000 cbBTC |
```
